docs/meetings/team/2022_11_01.md

```diff
@@ -0,0 +1,80 @@
+# 🚀 Weekly Team Meeting Minutes
+
+**Start Time** 9:36am
+
+**End Time** 10:45am
+
+**Location** Discord
+
+**Date** Nov 01 2022
+
+## 👋 Attendence
+
+1. Veronica
+2. Matt
+3. Edouard
+
+***Attendence Comments***
+
+Logan Parker could not make the meeting because is having a surgery today.
+
+## 🧸 Check-In
+
+## Announcements
+
+- GitStats due Friday
+
+- Gannt Chart due Friday
+
+- No Client Meeting
+  - Email Client
+
+## Last Week (Restrospective)
+
+- All tasks were completed
+
+- Veronica did some testing with Kuk
+
+- Matt and Ed were able to research Rhubarb integration
+
+- Logan got the local env set-up
+
+## This Week  
+
+### Gantt Chart
+
+- Veronica will upload on Friday
+
+### Rhubarb Peer Testing Process
+
+1) Naz could potentially provide us with a front-end for peer testing
```

Comment:
Their message looked like they can't provide us?

Reply:
no unfortunately they got nothing to provide :(